docs/get-started/using-standalone.md

```diff
@@ -11,6 +11,8 @@ Our [examples](https://github.com/uber/deck.gl/tree/master/examples/get-started)
 
 The [Deck](/docs/api-reference/deck.md) class takes deck.gl layer instances and viewport parameters, and renders those layers as a transparent overlay.
 
+You must create a canvas inside the html with the id `my-deck-canvas` for it to work.
```

Comment:
Move this comment close to line `29` to avoid confusion

Reply:
Done